spec/integration/publishing_a_scheduled_edition_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+RSpec.describe "Publishing a scheduled edition" do
```

Comment:
Yay - does this mean we can lose https://github.com/alphagov/content-publisher/pull/1140/commits/be778ef0181346c907d9a31fddd719dc390f38c7#diff-a7f748b67cad60e7c2600e6be3d0896b, since it should be covered by this test?

Reply:
I'm also wondering if we can lose the mailer spec, or maybe we should create one for the PublishMailer and simplify the associated feature specs.  Not part of this PR really, but it would be helpful if we used the same approach for each :-).

Reply:
I'll see if that test can be killed. 

I'd suggest the approach of having a PublishMailer test and loosen feature specs as there is quite a lot of conditionals and translation keys involved in the mails.

Reply:
👍 - should we have a follow-up task to make the approaches consistent?

Reply:
Yup, I've got that on my list that I need to move to trello.